src/pages/ProductDetail/ProductDetailInfo/Button.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const Button = ({ children, ...props }) => {
+  return <StyledButton {...props}>{children}</StyledButton>;
+};
+const StyledButton = styled.button`
+  display: flex;
+  font-weight: 700;
+  border: none;
+  width: 48%;
+  background-color: ${props => props.theme.redColor};
+  color: white;
+  padding: 20px;
+  border-radius: ${props => props.theme.btnRadius};
+
+  ${props =>
+    props.cell &&
+    css`
+      background-color: ${props => props.theme.greenColor};
+    `}
```

Comment:
```suggestion
  ${({cell, theme}) =>
    css`
      border-radius: ${theme.btnRadius};
      background-color: ${cell ? theme.greenColor : theme.redColor}
    `}
```
위와 같이도 표현할 수 있습니다~!!

Reply:
다른 부분도 응용할 수 있는 부분 응용해보세요 :)

Reply:
넵~